ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/AbstractRecord.java

```diff
@@ -52,6 +52,9 @@ public abstract class AbstractRecord extends StateManager
 
 	public abstract int typeIs ();
 
+	public void clearCache() {
```

Comment:
I think we should add a Javadoc for this. And how about calling it clearCachedType()? Cache might be a bit too broad?

Reply:
Agreed

Reply:
Although it is up to the various record types to decide what cached data they need to clear so clearCachedType is too restrictive.